### DISCUSSION TITLE
Unity WebcamTexture limitation

### DISCUSSION THREAD

**Discussion by QuentinBrun** [asus4/tf-lite-unity-sample#330] [General]
Hi,

I was wondering if any of you find the Unity WebCamTexture too limiting. Do you know of any other alternative ? Would accessing the native camera to better control the camera (& turn on the flash light on Android for example) be of any value ?

Thanks

**Comment by asus4** [comment]
Hi @QuentinBrun,

I have tried [NatDevice](https://docs.videokit.ai/natdevice/api/cameradevice#flash-mode), which supports camera control APIs. I think the [free subscription plan](https://www.natml.ai/videokit) includes these APIs. 

Please let me know if you find any other alternatives 👍
